### PR TITLE
DCS-2016 link name in recent arrival to prisoner summary

### PR DESCRIPTION
**Pull request by GurnankCheema**

### CODE REVIEW

File: integration_tests/pages/recentArrivals/prisonerSummary.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Page, { PageElement } from '../page'
+
+export default class PrisonerSummaryPage extends Page {
+  constructor(title, hasBackLink, hasFeedbackBanner) {
+    super(title, hasBackLink, hasFeedbackBanner)
```
Comment (andrewrlee) [inline integration_tests/pages/recentArrivals/prisonerSummary.ts:5]: you don't need to have parameters for `hasBackLink, hasFeedbackBanner` as they don't change?

File: integration_tests/pages/recentArrivals/prisonerSummary.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Page, { PageElement } from '../page'
+
+export default class PrisonerSummaryPage extends Page {
+  constructor(title, hasBackLink, hasFeedbackBanner) {
```
Comment (andrewrlee) [inline integration_tests/pages/recentArrivals/prisonerSummary.ts:4]: missing type info `title: string`?

File: integration_tests/integration/recentArrivals/recentArrivals.cy.ts [hunk 3]
```diff
@@ -3,13 +3,15 @@ import Page from '../../pages/page'
 import Role from '../../../server/authentication/role'
```
Comment (andrewrlee) [inline integration_tests/integration/recentArrivals/recentArrivals.cy.ts:3]: there's a binary file [cypress/downloads/downloads.html](https://github.com/ministryofjustice/hmpps-welcome-people-into-prison-ui/pull/388/files#diff-707e98447df3abc4f9a958a39e41dbb76470f48fbaef89674200b6f06a0aadac). That presumably needs to be deleted and potentially git ignored?

File: integration_tests/pages/recentArrivals/prisonerSummary.ts (hunk 2 above)
Comment (GurnankCheema) [reply integration_tests/pages/recentArrivals/prisonerSummary.ts:4]: type added

File: integration_tests/pages/recentArrivals/prisonerSummary.ts (hunk 1 above)
Comment (GurnankCheema) [reply integration_tests/pages/recentArrivals/prisonerSummary.ts:5]: These were added as the summary page doesn't have a backlink yet and page.ts is set to true as default
`    private readonly backNavigationLink: BackLink = { hasBackLink: true },`

File: integration_tests/integration/recentArrivals/recentArrivals.cy.ts (hunk 3 above)
Comment (GurnankCheema) [reply integration_tests/integration/recentArrivals/recentArrivals.cy.ts:3]: Not sure how that got in and I missed it when I checked the PR.
Deleted

File: integration_tests/pages/recentArrivals/prisonerSummary.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    super(title, hasBackLink, hasFeedbackBanner)
+  }
+
+  static goTo(prisonId: string, title: string, hasBackLink: boolean, hasFeedbackBanner: boolean): PrisonerSummaryPage {
```
[the remark this answers is not in the document]
Comment (GurnankCheema) [reply integration_tests/pages/recentArrivals/prisonerSummary.ts:8]: goTo() deleted

File: integration_tests/integration/recentArrivals/recentArrivals.cy.ts [hunk 5]
```diff
@@ -108,4 +114,72 @@ context('A user can view all recent arrivals', () => {
...
+        numberOfBodyScansRemaining: 100,
+      },
+    })
+
+    cy.signIn()
+    const recentArrivalsPage = RecentArrivalsPage.goTo()
+    recentArrivalsPage.recentArrivals(1, today).name().click()
+
+    const prisonerSummaryPage = new PrisonerSummaryPage(`${recentArrival.lastName}, ${recentArrival.firstName}`, {
+      hasBackLink: false,
```
Comment (andrewrlee) [inline integration_tests/integration/recentArrivals/recentArrivals.cy.ts:134]: don't need hasBackLink as a parameter either do you?

Comment (GurnankCheema) [reply integration_tests/integration/recentArrivals/recentArrivals.cy.ts:134]: agreed

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**